api/migrations/0010_auto_20180305_1252.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.0.1 on 2018-03-05 12:52
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0009_item_model_number'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='item',
+            name='model_number',
+            field=models.ForeignKey(
+                null=True, on_delete=django.db.models.deletion.PROTECT,
+                to='api.ItemModelNumber'),
+        ),
```

Comment:
@HawiCaesar ☝️

Reply:
True you did! When I was correcting the make model to include `models.PROTECT` I was deleting the migration file. So I have only one migration file instead of 2. But this works just as well.